src/org/labkey/snd/SNDManager.java

```diff
@@ -3514,14 +3783,18 @@ public Map<Integer, Event> getAllEvents(Container c, User u, List<Integer> event
         List<Event> events = eventSelector.getArrayList(Event.class);
         Collection<Map<String, Object>> eventsExtensibleFields = eventSelector.getMapCollection();
 
+        TableSelector eventDataSelector = getTableSelector(c, u, eventIds, SNDSchema.EVENTDATA_TABLE_NAME, "EventId", null, null);
```

Comment:
"EventId" = Event.EVENT_ID the column names are defined in the Event class in the API